### ISSUE TITLE
Skipping some code of train file

### ISSUE THREAD

**Issue by ghffarpk** [googlecolab/colabtools#233]
Hi
I am using colab to train a model. i have start the training and then abort because i found a error in data.
Now when i run it. it is starting from np.random.shuffle(utils.gt_labels) (line 51). Here is my code.

> import settings
import model
from utils import VOC
import os
import time
import tensorflow as tf
os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'
import numpy as np

device_name = tf.test.gpu_device_name()
if device_name != '/device:GPU:0':
  raise SystemError('GPU device not found')
print('Found GPU at: {}'.format(device_name))

#tf.reset_default_graph() 
sess = tf.InteractiveSession()
model = model.Model()
utils = VOC('train')
saver = tf.train.Saver(var_list = tf.get_collection(tf.GraphKeys.TRAINABLE_VARIABLES, scope = 'yolo'))
sess.run(tf.global_variables_initializer())

try:
    saver.restore(sess, "drive/app/test_yolo/model.ckpt")
    print "load model.."
except:
    print 'start from fresh variables'
            
for i in xrange(settings.epoch):
        
    last_time = time.time()
    total_loss = 0
    
    for x in xrange(0, len(utils.gt_labels) - settings.batch_size, settings.batch_size):
        images = np.zeros((settings.batch_size, settings.image_size, settings.image_size, 3))
        labels = np.zeros((settings.batch_size, settings.cell_size, settings.cell_size, 6))
        
        for n in xrange(settings.batch_size):
            imname = utils.gt_labels[x + n]['imname']
            flipped = utils.gt_labels[x + n]['flipped']
            images[n, :, :, :] = utils.image_read(imname, flipped)
            labels[n, :, :, :] = utils.gt_labels[x + n]['label']
            
        #print images.shape
        loss, _ = sess.run([model.total_loss, model.optimizer], feed_dict = {model.images: images, model.labels: labels})
        total_loss += loss
        
        #print loss
        if (x + 1) % settings.checkpoint == 0:
            print 'checkpoint reached: ' + str(x + 1)
    
      np.random.shuffle(utils.gt_labels)
      print 'epoch: ' + str(i + 1) + ', loss: ' + str(loss / (len(utils.gt_labels) - settings.batch_size / (settings.batch_size * 1.0))) + ',  s / epoch: ' + str(time.time() - last_time)
      saver.save(sess, 'drive/app/test_yolo/model.ckpt')

**When i comment the below code then it is starting the execution from line 1.**

**Comment by metrizable** [comment]
This appears to be an issue outside of Colab's control. We recommend asking on Stack Overflow.
